packages/block-library/src/template-part/edit/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * WordPress dependencies
```

Comment:
Are we going to support the use case of referring the same template part multiple times in a post?
When I tried to do it the block controls and selection appear multiple times.
![image](https://user-images.githubusercontent.com/11271197/71296354-2cec8580-2377-11ea-86b9-b2ee6de58b97.png)

Reply:
Good question. Template parts can be inserted multiple times: https://developer.wordpress.org/reference/functions/get_template_part/

Reply:
Good question. Template parts can be inserted multiple times: https://developer.wordpress.org/reference/functions/get_template_part/

Reply:
Yes, we need to support this.

This is happening because the inner blocks have the same client IDs. I am pretty sure this wasn't a problem before the recent selection refactoring and inclusion in undo history, so we should be able to fix it pretty easily.

@ellatrix will have a better idea of what to do since she worked on that, but I don't think this is a blocker for this PR.